From: dnf-for-termux <dev@termux.dev>
Date: Fri, 31 Jul 2026 00:00:00 -0600
Subject: [PATCH 9/10] Disable -Werror for Termux build

dnf5 enables -Werror for all sources, which fails on Termux/Android
due to warnings in the NDK toolchain headers and deprecation warnings
from older glibc/bionic interfaces. Drop the -Werror flag while
keeping -Wall -Wextra so warnings are still shown but non-fatal.

diff --git a/CMakeLists.txt b/CMakeLists.txt
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -84,4 +84,4 @@
 # warnings
-add_compile_options(-Wall -Wextra -Werror)
+add_compile_options(-Wall -Wextra)  # -Werror disabled for Termux build
 add_compile_options(-Wcast-align -Wformat-nonliteral -Wmissing-format-attribute -Wredundant-decls -Wsign-compare -Wsign-conversion -Wtype-limits -Wuninitialized -Wwrite-strings)
 add_compile_options(-Werror=unused-result -Wodr)
